### DISCUSSION TITLE
Adding backward compatibility for column accessor

### DISCUSSION THREAD

**Discussion by roman-khasimyanov** [TanStack/table#3710] [Ideas]
In v7 react-table column declaration has accessor with dot notation like `'field.subfield'`.
In v8 this no longer supported.

**Comment by tannerlinsley** [comment]
I don’t plan on adding this feature back. It caused a lot of issues beforehand and now with Typescript its a pretty bad experience. You can always use a utility to do this if you prefer.

**Reply by kblcuk** [reply]
Should update the docs then, according to them one can still use dot notation https://react-table.tanstack.com/docs/api/useTable#column-options

**Comment by tannerlinsley** [comment]
These docs are for v7 so that’s accurate. The new docs will reject this change.

Tanner Linsley
